### DISCUSSION TITLE
Better file picker

### DISCUSSION THREAD

**Discussion by Qronikarz** [linuxmint/discussions#513] [Ideas]
Original post for more info:
https://github.com/linuxmint/mint22-beta/issues/115

**Comment by JosephMcc** [comment]
This is outside of the scope of Mint really. The file choosers are supplied by whatever toolkit was used to create the application. Which means most of the time either Qt or Gtk. There was really no reason to reopen this here.

**Reply by Qronikarz** [reply]
> This is outside of the scope of Mint really.

Maybe it was out of the scope for Mint 22, but it's still not a solved problem for Mint.

It was a bug/report that has stayed open for 18 years. Linux Mint wants to stay on GTK3 which is understandable, but then it misses some things.

Now let's check which file choosers are supplied:
Firefox (GTK, native package) - Mint file picker
Steam (custom, Flathub) - Mint file picker
Bottles (GTK4+Libadwaita, Flathub) - Mint file picker
Audacious (Qt, Flathub) - Mint file picker
Audacious (Qt, native package) - Mint file picker
Krita (Qt, Flathub) - Qt file picker
KolourPaint (Qt, native package) - Qt file picker
Footage (GTK4+Libadwaita, Flathub) - Mint file picker
Amberol (GTK4+Libadwaita, Flathub, Gnome Circle) - Mint file picker

Also for 2 of the Qt pickers that I stumbled onto they both use my accent color in different manner.
This one uses it for selection, but folders stay in normal yellow:
![kolourpaint](https://github.com/linuxmint/discussions/assets/84787215/a1207acf-37dc-482d-a15f-3006d5f8f514)
This one has folders with my accent color, but selection is blue:
![krita](https://github.com/linuxmint/discussions/assets/84787215/f4bf725d-66bd-4e5e-9ba3-50e5d33593af)

And another point is that even the Qt file pickers aren't really any better. One of them even doesn't have the preview.

> There was really no reason to reopen this here.

So I don't feel like this idea should stay closed.
